matplotlib_scalebar/scalebar.py

```diff
@@ -496,16 +507,21 @@ def _get_value(attr, default):
         else:
             child = scale_box
 
-        box = AnchoredOffsetbox(
-            loc=location, pad=pad, borderpad=border_pad, child=child, frameon=frameon
-        )
-
-        box.axes = ax
-        box.set_figure(self.get_figure())
-        box.patch.set_color(box_color)
-        box.patch.set_alpha(box_alpha)
-        box.draw(renderer)
-
+        if self.bbox_to_anchor != None:
+            self.box = AnchoredOffsetbox(loc=location, pad=pad,  child=child, frameon=frameon, 
```

Comment:
This comment in https://github.com/ppinard/matplotlib-scalebar/issues/39#issuecomment-839988892 seem to suggest that the location shouldn't be used. Should it then be removed from the AnchoredOffsetbox?